docs/basics/101-122-config.rst

```diff
@@ -295,6 +295,21 @@ remaining sections in that file, and the :ref:`that dissects this config file fu
    Lastly, the ``annex-ignore = false`` configuration allows git-annex
    to query the remote when it tries to retrieve data from annexed content.
 
+.. index::
+   pair: configuration; DataLad command
+   pair: setting configurations; with DataLad
+
+The ``datalad configuration`` command
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Although this section put a focus on the ``git config`` command, it is important to mention that there also is a ``datalad configuration`` command.
+It is not identical to ``git config``, but while it lacks some feature of ``git config``, such as the ability to set system-wide configuration, it has additional features:
```

Comment:
```suggestion
It is not identical to ``git config``, but while it lacks some feature of ``git config``, such as the ability to set system-wide configuration, it has additional features.
```

I am removing such colons. To me they only serve a purpose within a sentence, but less so across.